context/app/templates/pages/ccf-eui.html

```diff
@@ -2,21 +2,9 @@
 <html lang="en">
 
 <head>
-  <script async src="https://www.googletagmanager.com/gtag/js?id=UA-133341631-3"></script>
```

Comment:
FYI we removed the google analytics tags above as they are now handled internal to the EUI